prez-components/src/components/PrezUINode.vue

```diff
@@ -1,34 +1,90 @@
 <script lang="ts" setup>
-import type { PrezUINodeProps } from '../types';
+import { RouterLink } from "vue-router";
 import Chip from "primevue/chip";
+import { PrezUINodeProps } from "../types";
 
 const props = defineProps<PrezUINodeProps>();
-
-const label = (props.term.label ? props.term.label.text :
-    (props.term.curie ? props.term.curie : props.term.uri));
-
```

Comment:
this logic for labels and tooltips covered a few specific scenarios that I tested with David in all of the scenarios in storybook previously. Are these scenarios similarly handled now?